package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "sfmovies",
+  "version": "0.0.0",
+  "description": "A RESTful API for movies filmed in San Francisco",
+  "main": "lib/index.js",
+  "repository": "git@github.com:pallavi-s/sfmovies",
+  "author": "pallavi@lob.com",
+  "license": "MIT",
+  "devDependencies": {
+    "chai": "^3.0.0",
+    "eslint": "^1.9.0",
+    "eslint-config-lob": "^2.0.0",
+    "generate-changelog": "^1.0.0",
+    "istanbul": "^0.4.2",
+    "mocha": "^2.2.5",
+    "nodemon": "^1.11.0"
+  },
+  "scripts": {
+    "enforce": "istanbul check-coverage --statement 100 --branch 100 --function 100 --lines 100",
+    "lint": "eslint .",
+    "release:patch": "changelog -p && git add CHANGELOG.md && git commit -m 'updated CHANGELOG.md' && npm version patch && git push origin && git push origin --tags",
+    "release:minor": "changelog -m && git add CHANGELOG.md && git commit -m 'updated CHANGELOG.md' && npm version minor && git push origin && git push origin --tags",
+    "release:major": "changelog -M && git add CHANGELOG.md && git commit -m 'updated CHANGELOG.md' && npm version major && git push origin && git push origin --tags",
+    "start": "nodemon --ignore test/",
+    "test": "NODE_ENV=test istanbul cover _mocha -- test --require test/setup.js --recursive --timeout 30000",
+    "test:raw": "NODE_ENV=test mocha test --require test/setup.js --recursive --timeout 30000"
+  },
+  "optionalDependencies": {
+    "fsevents": "^1.1.2"
```

Comment:
you should be able to just run `yarn remove fsevents` and not include it anywhere in the `package.json` and things should work